src/main.rs

```diff
@@ -31,7 +32,28 @@ fn delete_branches(branches: Vec<String>) -> i32 {
     count
 }
 
+// Windows Terminal renders U+2714 "Heavy Check Mark" as a purple emoji which masks the terminal
+// color and makes it impossible to determine if an item is selected or not.
+//
+// See: https://github.com/microsoft/terminal/issues/15592
+//      https://github.com/microsoft/terminal/issues/13110
+//
+// Fortunately, this doesn't affect U+2713 "Check Mark" so for Windows terminal we use that
+// character instead until the emoji issue is fixed.
+const WINDOWS_TERMINAL_SYMBOLS: SymbolSet = SymbolSet {
+    completed: '\u{2713}', // '✓'
+    ..requestty::symbols::UNICODE
+};
+
+fn is_windows_terminal() -> bool {
+    std::env::var("WT_SESSION").is_ok()
```

Comment:
@birtles 

If the window terminal is the default terminal, `WT_SESSION` returns null.
Does this work on your PC?

https://github.com/microsoft/terminal/issues/13006#issue-1221666658

Reply:
Oh, I didn't know that. My default terminal is "Windows Terminal (preview)" and `WT_SESSION` is set for me.

I tried setting my default terminal to "Windows Terminal" (not preview) and running it but `WT_SESSION` is still set for me. Maybe it's fixed?

Reply:
@birtles 
Could be fixed.
I will see when same problem happen next time in different environment!!

Thanks!